webbpsf/gridded_library.py

```diff
@@ -155,7 +155,7 @@ def __init__(
             from photutils.psf import GriddedPSFModel
         except ImportError:
             try:
-                from photutils import GriddedPSFModel
+                from photutils import GriddedPSFModel # noqa
             except ImportError:
                 raise ImportError('This method requires photutils >= 0.6')
```

Comment:
This nested import check can probably be simplified or dropped, since we're now requiring photutils >1.10 in pyproject.toml

Reply:
moving to top